instrumentation/github.com/gorilla/mux/otelmux/internal/semconvutil/httpconv.go

```diff
@@ -407,6 +407,7 @@ func (c *httpConv) ServerRequest(server string, req *http.Request) []attribute.K
 
 	if target != "" {
 		attrs = append(attrs, c.HTTPTargetKey.String(target))
+		attrs = append(attrs, c.HTTPTargetKey.String(target))
```

Comment:
This is using `HTTPTargetKey`, so it's setting the `http.target` attribute twice, not `http.route`.